XRTK-Core/Packages/com.xrtk.core/Definitions/Utilities/TrackedObjectType.cs

```diff
@@ -16,6 +16,10 @@ public enum TrackedObjectType
         /// <summary>
         /// Calculates position and orientation from the right motion-tracked controller.
```

Comment:
Right controller, not just mr motion controller

Reply:
That's a breaking change but sure